source/renderer/app/components/staking/delegation-center/DelegationCenterBody.scss

```diff
@@ -17,7 +17,32 @@
 
   .bodyTitle {
     @extend %regularLightColorText;
-    margin: 0 0 10px 20px;
+    display: flex;
+    margin: 0 0 10px 0;
+    padding: 0 20px;
+
+    .leftBodyTitle {
+      flex: 2.5;
+    }
+
+    .rightBodyTitle {
+      display: flex;
+      flex: 1.5;
+      justify-content: flex-end;
+
+      > span:first-child {
+        margin: 0 52px;
```

Comment:
Please avoid using margins for text centering 🙏

Reply:
Fixed